angle_gains_init: [0.4, 0.0, 0.0]
angle_gains_final: [0.4, 0.01, 0.05]
dist_gains: [0.125, 0.0, 0.0] #0.25
dock_backwards: true
angle_tolerance: 0.0436332 
dist_tolerance: 0.01
dist_feasible: 2.0
